### ISSUE TITLE
Schedule times shifted by one hour on start of daylight savings time

### ISSUE THREAD

**Issue by Moonbase59** [AzuraCast/AzuraCast#6177]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #5963965 (2023-03-23 0:28)

### What happened?

In Germany, we just had the shift to DST (daylight savings time, from "CET" to "CEST"). On Sunday 2023-03-26, at 02:00 AM local time, we switched to 03:00 AM.

The public **schedule page now shows all times off by +1 hour**, i.e. "7:00" instead of "6:00", see "Sunday" on the right:

![Auswahl_201](https://user-images.githubusercontent.com/3706922/227754794-492f7b9f-fd05-4462-9aed-289d10fdf0a8.png)

Playlists, nevertheles, play at the correct times, and show up as playing "now" in the station’s profile view.

I tried with both server timezone _Etc/UTC_ and _Europe/Berlin_. Same result. System works correctly but times shown on schedule page off by +1 hour.

### Relevant log output

```shell
—
```

**Comment by netbladenl** [comment]
Same issue here with the latest rolling release.

**Comment by Moonbase59** [comment]
Since all else works well, and playlists actually _play_ at the correct times, I suspect some small oversight in the schedule display code, probably easy to fix.

**Comment by guiltlab** [comment]
Once again, this seems to have fixed itself. No restart of azuracast/server. It took about a week. I tried to clear cache in Firefox a bunch of times but didn't see any difference, so I'm not sure what made it correct again.

**Comment by BusterNeece** [comment]
I think this happens because weeks start on Sunday on the calendar, and that would mean they start just a few hours _before_ DST kicks in on the week that DST kicks in, so all the times are 'wrong' for that whole week.

I also recall something similar the last time the DST times switched around.

**Comment by robz0** [comment]
I noticed that the while azuracast app had the right time set, the Server VM did not, it was an hour earlier.
I used ```timedatectl set-timezone``` to set the server time but the docker instance is still an hour out. But I cannot use that command inside there. 

I noticed the time was out, while looking at logs, which was posting the earlier time, which was not the time shown in AzuraCast Ui. 

Is there a way or a need to set the zone inside the docker. 

We are trying to debug scheduling issues.
Thanks
